lib/internal/util.js

```diff
@@ -101,13 +104,30 @@ function deprecate(fn, msg, code) {
   return deprecated;
 }
 
-function decorateErrorStack(err) {
+function decorateErrorStack(err, options) {
   if (!(isError(err) && err.stack) ||
       getHiddenValue(err, kDecoratedPrivateSymbolIndex) === true)
     return;
 
   const arrow = getHiddenValue(err, kArrowMessagePrivateSymbolIndex);
 
+  if (options?.format === 'commonjs' && arrow) {
+    const {
+      childSpecifier
+    } = options;
+    const importStatement = StringPrototypeSplit(arrow, '\n')[1];
+    const namedImports = StringPrototypeMatch(importStatement, /{.*}/)[0];
```

Comment:
```suggestion
    const namedImports = StringPrototypeMatch(importStatement, /{.*}/)?.[0];
```
this would avoid a crash if the match is null, even if you're sure it could never happen :-)

Reply:
We wouldn't be able to do this in v12.x

Reply:
I really don't think we should be adding this additional check if it is not needed.